jest.config.js

```diff
@@ -1,11 +1,12 @@
 module.exports = {
   clearMocks: true,
-  moduleFileExtensions: ['js', 'ts'],
+  moduleFileExtensions: ['ts', 'js'],
```

Comment:
Switched these to give a priority of typescript files over js when looking for tests.

Reply:
Could you add a comment explaining why it's necessary? Could we drop js altogether?